### PR TITLE
Translated "SSH" and "SSH key" to Ukrainian

### PR DESCRIPTION
**Pull request by Dod900ls1**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Yehor Boiar 

## Language: 

- Ukrainian

## Terms defined:

- SSH
- SSH key

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -9350,7 +9356,12 @@
       A string of random bits stored in a file that is used to identify a user for
       [SSH](#ssh). Each SSH key has separate public and private parts; the public part
       can safely be shared, but if the private part becomes known, the key is compromised.
-
+  uk:
+    term: "SSH-ключ"
+    def: >
+      Рядок випадкових бітів, збережений у файлі, який використовується для ідентифікації
+      користувача в SSH. Кожен SSH-ключ має окремі публічну та приватну частини; публічну
+      частину можна безпечно ділитися, але якщо приватна частина стає відомою, ключ вважається скомпрометованим.
```
Comment (Herobread) [inline glossary.yml:9364]: Заміни [ділитися](https://slovnyk.ua/index.php?swrd=%D0%B4%D1%96%D0%BB%D0%B8%D1%82%D0%B8%D1%81%D1%8F) на [поширювати](https://slovnyk.ua/index.php?swrd=%D0%BF%D0%BE%D1%88%D0%B8%D1%80%D1%8E%D0%B2%D0%B0%D1%82%D0%B8)

Ділитися означає split, a поширювати - share

**Review by Herobread — CHANGES_REQUESTED** [review]

File: glossary.yml (hunk 1 above)
Comment (Dod900ls1) [reply glossary.yml]: Changed

**Review by Herobread — APPROVED** [review]

**Comment by ccoVeille** [comment]
Hi @Dod900ls1 

I'm a random guy who watches the repository.

While I see as a good thing that you are adding translations, I'm surprised you are not grouping them.

I mean you are adding them one by one, every few days.

**Comment by ccoVeille** [comment]
I mean, this is a bit noisy

**Comment by Dod900ls1** [comment]
@ccoVeille Good point.

**Comment by olexandr-konovalov** [comment]
@ccoVeille sorry for lots of notifications - we have a translation sprint in June-July, so there may be quite a few more coming shortly. Please note this if you'd like to adjust your watching preferences for these two months. The repository has both examples of PR for a single word and for multiple terms, and Glosario maintainers are happy with us doing it that way. OTOH, we plan to review and merge PRs quickly, so they will disappear from the list of open PRs, reducing the noise.

One thought that I have against mixing many words in one PR - if you have commit A for word X, and then later commit B for word Y, you can't fix the commit A by `--amend` anymore, as it's not the latest. You can of course, use `git rebase -i` to refactor the history, and rewrite older commits, but that require more advanced skills - not every contributor is well versed with these commands yet...

**Comment by ccoVeille** [comment]
Thanks for giving context in your detailed reply.

I think I will adjust my settings, yes

File: glossary.yml [hunk 2]
```diff
@@ -9350,7 +9356,12 @@
       A string of random bits stored in a file that is used to identify a user for
       [SSH](#ssh). Each SSH key has separate public and private parts; the public part
       can safely be shared, but if the private part becomes known, the key is compromised.
-
+  uk:
+    term: "SSH-ключ"
+    def: >
+      Рядок випадкових бітів, збережений у файлі, який використовується для ідентифікації
+      користувача в SSH. Кожен SSH-ключ має окремі публічну та приватну частини; публічну
```
Comment (olexandr-konovalov) [inline glossary.yml:9363]: @Dod900ls1 you can add hyperlink to `[SSH](#ssh)` like in English text

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]

**Review by olexandr-konovalov — APPROVED** [review]
@elletjies please merge this too! thanks!

**Pull request merged**